delta-inputs/s3-to-stash.ts

```diff
@@ -41,8 +65,44 @@ s3Stream.on('finish', () => process.exit(0))
 const deltaWriteStream = stashDB.deltaWriteStream()
 deltaWriteStream.on('deltaProcessed', () => tsvRowCount++)
 
+class SetSelfVesselId extends Transform {
+  readonly selfId: string
+  readonly isSelf: (delta: any) => boolean
+  readonly importAll: boolean
+
+  constructor(selfVesselId: string, importAll: boolean | undefined) {
+    super({ objectMode: true })
+    this.selfId = 'vessels.' + selfVesselId
+    this.importAll = importAll !== undefined
+    this.isSelf =
+      selfVesselId !== undefined
```

Comment:
Is this ternary really needed? `selfVesselId` is typed for not being `undefined`, is required parameter by `commander` module and checked for matching either uuid or mmsi.